### PR TITLE
Add local databases

### PR DESCRIPTION
**Pull request by CharlesNepote**
Some files are local databases created by the app on each environment: *.sto, top_translators.csv. They have to be ignored by git.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,9 @@ logs/modperl_error_log
 
 # Local databases
 data/mongodb
+Lang.openfoodfacts.*
+users_emails.sto
+html/data/top_translators.csv
```
Comment (hangy) [inline .gitignore]: Could we ignore html/data/ as a whole? There's other scripts that export MongoDB, RDF, etc. there.

File: .gitignore [hunk 2]
```diff
@@ -20,6 +20,9 @@ logs/modperl_error_log
 
 # Local databases
 data/mongodb
+Lang.openfoodfacts.*
```
Comment (hangy) [inline .gitignore]: Is `openfoodfacts` static or could it also be `openpetfoodfacts`?

File: .gitignore (hunk 1 above)
Comment (CharlesNepote) [reply .gitignore]: Ok, done.

File: .gitignore (hunk 2 above)
Comment (CharlesNepote) [reply .gitignore]: I have put `Lang.open*`.

I think I will open a new bug to suggest moving this files from root directory.

**Comment by CharlesNepote** [comment]
As it's very anecdotic changes, I'll merge tomorrow if there are no more comments.

**Pull request merged**